src/gui/properties/trackerlistwidget.cpp

```diff
@@ -568,12 +568,12 @@ void TrackerListWidget::showTrackerListMenu(const QPoint &)
     }
 
     if (!torrent->isPaused()) {
-        const QAction *reannounceSelAct = menu->addAction(UIThemeManager::instance()->getIcon("view-refresh"), tr("Force reannounce to selected trackers"));
```

Comment:
Before you guys go all out, just keep in mind the original icon name has an advantage which is using the icon from desktop environment (if they provided one), this makes the application having more native feel. https://developer.gnome.org/icon-naming-spec/
IMO we should use those standard icon names as much as possible and only use custom ones when standard ones are not suitable.

Reply:
I didn't know that. Let me adjust using the icon specific name if possible.

Reply:
I have checked the gnome-icon-theme. There is nothing that can replace `reannounce` and `tag` icon. 
I am going to change the `recheck` icon to `view-refresh` as they are very similar. Great tip!